pages/api/exam-result.js

```diff
@@ -0,0 +1,50 @@
+import fs from "fs/promises";
+import examConfigs from "../../examConfig";
+
+export default async function handler(req, res) {
```

Comment:
1. Add a comment here -- few lines explaining what is does
   - Give example of URL with query parameters